readme.md

```diff
@@ -37,7 +38,7 @@ Yesterday afternoon you learned how to use Angular and `ngResource` to communica
 
```

Comment:
Add comma to line 33:
we need to provide a `url`, `type` (HTTP method) and `dataType`